src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -128,6 +128,11 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         filteredPersons.setPredicate(predicate);
     }
 
+    @Override
+    public void sortFilteredPersonList() {
+        this.addressBook.sortPersonList();
```

Comment:
I think the `this` keyword can be removed for consistency with the rest of the `ModelManager` class.

```suggestion
        addressBook.sortPersonList();
```